client/src/main/java/org/apache/uniffle/client/util/ClientUtils.java

```diff
@@ -122,4 +126,11 @@ public static void validateTestModeConf(boolean testMode, String storageType) {
               + "because of the poor performance of these two types.");
     }
   }
+
+  public static void validateClientType(String clientType) {
```

Comment:
Could we convert this method to `CheckValue#checkValueFunc` like `ConfigUtil#POSITIVE_LONG_VALIDATOR`?

Reply:
OK, i will try it.

Reply:
It doesn't seem to work because there is no `checkValue` on mr's client side.

Reply:
I got it. My mistake.